site/content/contribute/why-contribute/_index.md

```diff
@@ -0,0 +1,99 @@
+7. Mattermost has a system to categorize the inactivity of contributions. Invalid PRs don’t need to go through this cycle at the Community Coordinator’s discretion.
+    1. After 10 days of inactivity, a contribution becomes stale, and a bot will add the `lifecycle/1:stale` label to the PR. It’s on the Community Coordinator to nudge the right people to get a stale PR active again, even if that means clarifying requests so the contributor has more information to go off of.
+    2. After 20 days of inactivity, a contribution becomes inactive, and a bot will add the `lifecycle/2:inactive` label to the PR. The Community Coordinator warns everybody involved how much time they have before the contribution is closed and again tries to reach out to the blocking party to help. They’re also going to make sure that it’s not the reviewers taking the PR to this point — contributions should only ever be inactive because of no response from the contributor.
+        1. When contributions are inactive but for good reason (for example, when the team is actively discussing a major design decision but they haven’t decided on anything yet), `lifecycle/frozen` would be a better label.
+        2. Inactive contributions are eligible to be picked up by another community member.
+    3. After 30 days of inactivity, a contribution becomes orphaned, and they’ll add the `lifecycle/3:orphaned` label to the now-closed PR. The associated `Help Wanted` ticket is given back its `Up For Grabs` status so others can pick up on the issue.
+
+## Subsection 2 — You want to help with content
+
+1. Good content is just as important as good code! If you notice and fix a content error in the documentation or in another open source article describing Mattermost, we consider you to be a valued member of our contributor community just like if you had added a feature to the core code.
+2. If you see a problem with Mattermost [developer](https://developers.mattermost.com/) or [user](https://docs.mattermost.com/) documentation, you have a few options:
+    1. If you have time to fix the mistake and it only affects a single page, navigate to the applicable page and click “Edit in GitHub” at the top right.  It’ll walk you through the process of creating a fork so that you can then follow the steps under section 1, subsection 1 of this guide, “You’ve found a bug”.
+    2. If you don’t have time to fix the mistake, copy the file path you’re on and create a GitHub issue about the problem you found. Make sure to include the file path and fill out the issue template completely to maximize clarity.
+    3. If you’re not up for creating a GitHub issue right now, that’s alright too! In the bottom right corner of every docs page is the question “Did you find what you were looking for?” Use this to quickly provide direct feedback about any page you’re viewing.
```

Comment:
This only applies to the product docs at the moment. There is an existing request to add this functionality to the developer docs site. FYI.